packages/react-components/src/components/utils/GridLayoutUtils.ts

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
nit: I'd just keep these helper functions etc in `GridLayout.tsx`

They're all used by that file, and splitting into a Utils file doesn't add much value.